commands/browsing/convert-twitter-to-nitter.js

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env node
+
+// Required parameters:
+// @raycast.schemaVersion 1
+// @raycast.title Convert Twitter to Nitter
+// @raycast.mode silent
+
+// Optional parameters:
+// @raycast.icon 🐔
+
+// Documentation:
+// @raycast.description Convert Twitter link to Nitter
+// @raycast.author csharp
+// @raycast.authorURL https://raycast.com/csharp
+// @raycast.argument1 { "type": "text", "placeholder": "Twitter Link" }
+
+// Example Twitter link: https://twitter.com/Cron/status/1644010827647975425
+const child_process = require('child_process');
+let [link] = process.argv.slice(2);
```

Comment:
I honestly don't think you need to make use of arguments in here, since your user are probably going to copy the URL, you can simply copy it directly from the clipboard area improving the experience of those who will use your Script Command. 

What do you think of the following suggestion?

```suggestion

// Example Twitter link: https://twitter.com/Cron/status/1644010827647975425
const child_process = require('child_process');
let link = child_process.execSync("pbpaste").toString();
```

this way you will have one less thing to do when using it ;)